File: URI Online Judge/area.c
#include <stdio.h>
 
int main() {
 double a,b,c,areaOfTriangle,areaOfCircle,pi=3.14159,areaOfTrapezoid,areaOfSquare,areaOfRectangle;
 scanf("%lf",&a);
 scanf("%lf",&b);
 scanf("%lf",&c);
 areaOfTriangle=(a*c/2);
 areaOfCircle=(pi*pow(c,2));
 areaOfTrapezoid=(c*(a+b)/2);
 areaOfSquare=pow(b,2);
 areaOfRectangle=a*b;
 printf("TRIANGULO: %0.3lf\n",areaOfTriangle);
 printf("CIRCULO: %0.3lf\n",areaOfCircle);
 printf("TRAPEZIO: %0.3f\n",areaOfTrapezoid);
 printf("QUADRADO: %0.3lf\n",areaOfSquare);
 printf("RETANGULO: %0.3lf\n",areaOfRectangle);

 
    return 0;
}

File: URI Online Judge/Python/Area.py
a, b, c = [float(i) for i in input().split()]
pi=3.14159
areaTriangulo=(a*c)/2
areaCirculo=pi*(c*c)
areaTrapezio=c*(a+b)/2
areaQuadrado=(b*b)
areaRetangulo=a*b
print("TRIANGULO: %0.3f" % areaTriangulo)
print("CIRCULO: %0.3f" % areaCirculo)
print("TRAPEZIO: %0.3f" % areaTrapezio)
print("QUADRADO: %0.3f" % areaQuadrado)
print("RETANGULO: %0.3f" % areaRetangulo)